server/service/middleware/mdmconfigured/mdmconfigured.go

```diff
@@ -1,5 +1,3 @@
-// Package mdmconfigured implements a middleware that ensures that MDM is
-// configured.
```

Comment:
Was that on purpose? Would be good to have package doc, even if the package is relatively simple. (maybe now it's something like `implements middleware functions for the supported platform-specific MDM solutions to ensure MDM is configured and fail fast before reaching the handler if that is not the case`).